app/server/jest.config.js

```diff
@@ -2,6 +2,6 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
-  testTimeout: 120000,
+  testTimeout: 240000,
```

Comment:
oh dear :laughing:

Reply:
that was a test haha good catch I've reverted